Tests/App/config/config.yml

```diff
@@ -45,7 +45,11 @@ smartbox_integration_framework:
         amqp:
             type: AMQP
             description: AMQP queue driver
+            host: "%hostname%"
+            username: "%rabbitmq.username%"
+            password: "%rabbitmq.password%"
             format: json
+            vhost: "%rabbitmq.vhost%"
```

Comment:
Based on the implementation done, if I didn't see bad, the amqp connections are setup from the `connections` key, not from the new parameters you added like (host, username, password, vhost)

Reply:
No. The values here are used on initialization of the service by the compiler pass process. The other key connection is used only for unit tests in the case I want to simulate multiple connections.